hostapd/hostapd_cli_zephyr.c

```diff
@@ -269,7 +269,7 @@ static void hostapd_cli_recv_pending(struct wpa_ctrl *ctrl, struct hostapd_data
 			wpa_printf(MSG_DEBUG, "Received len: %d, msg_len:%d - %s->END\n",
```

Comment:
@GaofengZhangNXP Please change the commit subject to `zephyr: modify log type from info to debug`
now there the hostap mentioned twice which looks weird. As this is zephyr specific change, lets use the `zephyr: ...` prefix here.

Reply:
Hi @GaofengZhangNXP  please modify as jukkar's comment.
`zephyr: modify log type from info to debug`